docs/install/kubernetes/aws.md

```diff
@@ -54,7 +54,7 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 
 metadata:
-  name: flowforge
+  name: FlowFuse
```

Comment:
Just checking. Is this now current?

In an earlier part of the review, we still refer to AWS as using `flowforge` moniker

Reply:
Not sure this should be changed until we do other packaging changes - @hardillb to confirm.

Reply:
That is just the name of the cluster in AWS, it can be chnaged